GBShop/App/AppDelegate.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  AppDelegate.swift
+//  GBShop
+//
+//  Created by Igor Rais on 05.12.2021.
+//
+
+import UIKit
+
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate {
+    
+    let requestFactory = RequestFactory()
+    
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        let auth = requestFactory.makeAuthRequestFatory()
+        auth.login(userName: "Somebody", password: "mypassword") { response in
+            switch response.result {
+            case .success(let login):
+                print(login)
+            case .failure(let error):
+                print(error.localizedDescription)
+            }
+        }
+        auth.logout(idUser: 123) { response in
+            switch response.result {
+            case .success(let logout):
+                print(logout)
+            case .failure(let error):
+                print(error.localizedDescription)
+            }
+        }
+        let change = requestFactory.makeRegistrationAndChangePersonalDataFactory()
+        change.changePersonalData(idUser: 123, userName: "Somebody", password: "mypassword", userEmail: "some@some.ru", userGender: "m", userCreditCardNumber: "9872389-2424-234224-234", userBio: "This is good! I think I will switch to another language") { response in
```

Comment:
Please set data in a variable and send it change.changePersonalData as only parameter